website/cue/reference/glossary.cue

```diff
@@ -55,6 +55,17 @@ glossary: #Glossary & [
 			typically refers to the ability to retain data across restarts.
 			"""
 	},
+	{
+		term:        "Enrichment tables"
+		description: """
+			File-based information that you can use to enrich Vector [events](#event) in
```

Comment:
I think there are plans to potentially provide access to non-file-based information. Although I guess we could update this description if/when that happens.